packages/adaptive/README.md

```diff
@@ -0,0 +1,158 @@
+# `@gitbook/adaptive`
+
+GitBook supports integrations with popular feature flag providers like [LaunchDarkly](https://launchdarkly.com) and [Bucket](https://bucket.co), making it easy to personalize your documentation based on active feature flags.
```

Comment:
I wonder if we should be more general at the start of the readme when describing what the SDK does as it also exposes generic utils like `writeGitBookVisitorCookie` to allow devs to also write cookies with their own data in case they don't use LD or Bucket. It may also intro additional helpers in the future to be used on the backend side for writing signed cookies (not yet implemented).

Also we should may have the first paragraph introduce adaptive content (maybe link to the docs) and mention that this SDK provide utils to help facilitate passing data to GitBook for adapting their content.

And then add 2 sections:
- first one that talks specifically about FF helper
- another one that talks about the generic one `writeGitBookVisitorCookie`

@taranvohra what do you think?

Reply:
Added in this feedback